### PR TITLE
qt5-qtcreator: update version 4.6.0->4.6.1

### PR DESCRIPTION
**Pull request by MarcusCalhoun-Lopez**
###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.13
Xcode 9.3

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/qt5-qtcreator/Portfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PortGroup           cxx11  1.1
 
 name                qt5-qtcreator
 
-version             4.6.0
+version             4.6.1
 #revision            0
```
Comment (pmetzger) [inline devel/qt5-qtcreator/Portfile]: Why the commented out `revision`?

**Comment by pmetzger** [comment]
Other than the commented out `revision`, LGTM. It's your port so you should feel free to commit.

File: devel/qt5-qtcreator/Portfile (hunk 1 above)
Comment (MarcusCalhoun-Lopez) [reply devel/qt5-qtcreator/Portfile]: There is no compelling reason for the comment.
Whenever someone has `revision  0`, it is correctly pointed out that this is the default and should not be there.
However, I find it a very very minor inconvenience to retype `revision` when I want to increase the revision.

Comment (pmetzger) [reply devel/qt5-qtcreator/Portfile]: Normally we don't do this, but it's your port.

**Pull request merged**